pyfixest/FixestMulti.py

```diff
@@ -547,7 +547,7 @@ def confint(self) -> pd.Series:
             A pd.Series with coefficient names and confidence intervals.
             The key indicates which models the estimated statistic derives from.
         """
-        return self.tidy()[["2.5 %", "97.5 %"]]
+        return self.tidy()[["2.5%", "97.5%"]]
```

Comment:
this change breaks `coefplot` btw
![image](https://github.com/s3alfisc/pyfixest/assets/12086926/05021e9e-a095-4e36-bc46-cfda392327f4)